src/Shared/components/Loader.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { InfinitySpin } from "react-loader-spinner";
+
+const Loader = () => {
+  return (
+    <div className="loading-spinner">
+      <InfinitySpin color="rgb(246, 162, 67)" width="200" />
```

Comment:
avoid to use inline css

Reply:
can't assign classname or id to library component